agent_build_refactored/managed_packages/files/bin/agent-libs-config

```diff
@@ -52,6 +52,7 @@ def initialize():
             "pip",
             "install",
             "--quiet",
+            "--disable-pip-version-check",
```

Comment:
👍 

Probably not a bad idea for us to have some check in CI/CD which periodically reminds us to upgrade pip in the venv or similar.

Reply:
yeah, especially with the fact that we cache our builds.